app/image/image-details-directive.html

```diff
@@ -52,22 +52,27 @@
       <div class="form-group">
         <label class="col-sm-2 control-label"><span class="glyphicon glyphicon-qrcode"></span> ID</label>
         <div class="col-sm-10">
-          <p class="form-control-static"><a href="image/{{imageDetails.id}}">{{imageDetails.id | limitTo: 12}}</a></p>
+          <p class="form-control-static">
+          <!-- <a href="image/{{imageDetails.id}}">{{imageDetails.id | limitTo: 12}}</a> -->
```

Comment:
Please, remove code you commented out. This goes for probably a lot more places in your changes so I'm not going to say it for every change.